soccer/src/soccer/gameplay/gameplay_module.hpp

```diff
@@ -39,7 +39,7 @@ namespace Gameplay {
  * iteration of the main run loop, the GameplayModule
  * calls into python, which does all of the high-level planning, resulting in
  * updated motion targets, etc for the robots.  The
- * GameplayModule then executes path planning for each OurRobot.
+ * GameplayModule then executes path planning for each ::OurRobot.
```

Comment:
?

Reply:
oh woops I was testing to see why breathe didn't link `OurRobot` so I was messing around. I'll revert this.